.golangci.yml

```diff
@@ -17,4 +17,19 @@ run:
 linters-settings:
   gocritic:
     disabled-checks:
-      - singleCaseSwitch
\ No newline at end of file
+      - singleCaseSwitch
+
+issues:
+  exclude-rules:
+    - linters:
+        - staticcheck
+      text: "SA1019: package github.com/golang/protobuf/proto is deprecated" # ignore deprecation of proto library. We don't want to migrate yet because go-control-plane is not ready
+    - linters:
+        - staticcheck
+      text: "SA1019: package github.com/golang/protobuf/jsonpb" # ignore deprecation of proto library. We don't want to migrate yet because go-control-plane is not ready
+    - linters:
+        - staticcheck
+      text: "SA1019: proto.MessageName is deprecated" # ignore deprecation of proto library. We don't want to migrate yet because go-control-plane is not ready
+    - linters:
+        - staticcheck
+      text: "SA1019: proto.MessageType is deprecated" # ignore deprecation of proto library. We don't want to migrate yet because go-control-plane is not ready
```

Comment:
Shall we add `TODO` here so we can find such pieces in the code?

Reply:
added